scripts/converters/format_source.py

```diff
@@ -0,0 +1,32 @@
+import argparse
+import json
+from typing import List
+
+from linz_logger import get_log
+
+
+def format_source(source: List[str]) -> List[str]:
+    """Due to Argo constraints if using the basemaps cli list command
+    the source has a string that contains a list that needs to be split.
+    example: ["[\"s3://test/image_one.tiff\", \"s3://test/image_two.tiff\"]"]
+    """
+    if len(source) == 1 and source[0].startswith("["):
+        try:
+            source_json: List[str] = json.loads(source[0])
+            return source_json
+        except json.JSONDecodeError as e:
+            get_log().debug("Decoding Json Failed", source=source, msg=e)
```

Comment:
should this really swallow the error? If a invalid input is detected should it not throw?

Reply:
The way this change appears is really confusing. I haven't write this code. I just moved the file.

Reply:
I'll fix that in an other PR